Message/Message.py

```diff
@@ -0,0 +1,97 @@
+
+import hashlib
+import json
+
+from MessageTypes import Type
+
+class Message():
+    """Message class.
+
+    Args:
+        type (Type, optional): Type of the message. Defaults to None.
+        status (int, optional): Returning status. Defaults to None.
+        payload (str, optional): Sent data. Defaults to None.
+    """
+        
```

Comment:
Please remove those trailing whitespaces. I recommend this extension:
https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

Similar elsewhere.